### PR TITLE
Homework tools#2

### PR DESCRIPTION
**Pull request by piezol**

### CODE REVIEW

**Review by Kasperice — APPROVED** [review]
Solution looks good

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Iinc/ -Wall -Wextra -pedantic -Werror -o main
```
Comment (DamiXioR) [inline Makefile:2]: in my opinion include directory flag looks better in this way: -I./inc

**Review by DamiXioR — COMMENTED** [review]
nice work

File: src/functions.cpp [hunk 2]
```diff
@@ -1,9 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"
 #include <iostream>
 #include <limits>
+#include <algorithm>
```
Comment (Marcinati) [inline src/functions.cpp:4]: Sort headers.

**Review by Marcinati — CHANGES_REQUESTED** [review]

File: tutor [hunk 3]
```diff
@@ -0,0 +1,967 @@
...
+
+  1. Move the cursor to the line below marked --->.
+
+  2. Type  :s/thee/the <ENTER>  .  Note that this command only changes the
+     first occurrence of "thee" in the line.
+
+  3. Now type  :s/thee/the/g .  Adding the  g  flag means to substitute
+     globally in the line, change all occurrences of "thee" in the line.
+
+---> thee best time to see thee flowers is in thee spring.
```
Comment (ziobron) [inline tutor:564]: missing lesson ;)

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**